filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -189,6 +189,7 @@ void OpenGLContext::initExtensionsGLES(GLint major, GLint minor, ExtentionSet co
     ext.texture_compression_s3tc = hasExtension(exts, "WEBGL_compressed_texture_s3tc");
     ext.EXT_multisampled_render_to_texture = hasExtension(exts, "GL_EXT_multisampled_render_to_texture");
     ext.KHR_debug = hasExtension(exts, "GL_KHR_debug");
+    ext.EXT_texture_sRGB = hasExtension(exts, "GL_EXT_texture_sRGB");
```

Comment:
`EXT_texture_sRGB` is a not a GL ES extension. The equivalent in GL ES is [`EXT_texture_sRGB_decode`](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_sRGB_decode.txt).

Reply:
Thanks for catching this.  I think the one we want is: GL_EXT_texture_compression_s3tc_srgb, which I have now added.